### ISSUE TITLE
Availability germline V(D)J gene lists for Upadhyay dataset

### ISSUE THREAD

**Issue by SergeWielhouwer** [tAndreani/scBCR#6]
Hello Tommaso, 

Thank you for your work on assessing various Ig receptor reconstruction methods.

Do you perhaps still have ground truth tables Heavy_Chains_igblast.fmt7 and Light_Chains_igblast.fmt7 available? I am currently not fully sure whether my own excel sheets (Through TRUST4 annotator) matches all the correct germline genes to use for determining sensitivity scores. With these tables, I am completely sure that the hits from the Sanger sequences match with the used Igblast database version at that time.

E.g. Heavy chain list I am using now.
![image](https://user-images.githubusercontent.com/87863674/202198468-ea8870c9-d681-4665-ad4f-adfe218de1ff.png)

Looking forward to your reply.

Regards,

Serge

**Comment by tAndreani** [comment]
Hello Serge,  

thanks for your message. If you have immcantation or igblast installed you can annotate the ground truth of Upadhyay by using the fasta sequence in this github repository.

 **I am sending you the link of it: https://github.com/tAndreani/scBCR/tree/main/Real_Data/Ground_Truth/Upadhyay**. You can also find the script to obtain from the Heavy and Light chain fasta sequence the VDJ annotation, CDR3, productivity etc... **Furthermore, you could directly consult the paper at the figure 2 C** to have an idea to what expect on the comparison you have in mind (based on gene level comparison). 

Generally, TRUST4 and MiXCR have been reported for this dataset to have a D gene mis-hit problem. For more info you could also check this closed issue where me and the developer of TRUST4 discussed exactly this: https://github.com/tAndreani/scBCR/issues/5.

Please let me know

**Comment by SergeWielhouwer** [comment]
Hi Tommaso,

Thank you for your quick response and explanation.

I have already found the FASTA's and shell script from the link you have sent me. I did use those FASTAs using TRUST4 annotator already, thanks for pointing out the TRUST4/MiXCR issue! Very valuable information. I will definitely try out annotating the ground truth using Igblast 1.19.0 with the latest IMGT germline sequences to see what genes will be reported. (I find it easier to use than immcantation for now).
